packages/governance/test/swingsetTests/contractGovernor/test-governor.js

```diff
@@ -98,11 +98,49 @@ const expectedcontractGovernorStartLog = [
+  'Voter Dave voted for {"changeParam":{"key":"main","parameterName":"Electorate"},"proposedValue":{"brand":"[Alleged: Zoe Invitation brand]","value":[{"description":"questionPoser","expiry":"[undefined]","fee":"[undefined]","handle":"[Alleged: InvitationHandle]","installation":"[Alleged: Installation]","instance":"[Alleged: InstanceHandle]","zoeTimeAuthority":"[undefined]"}]}}',
+  'Voter Emma voted for {"noChange":{"key":"main","parameterName":"Electorate"}}',
+  '@@ tick:1 @@',
+  '@@ tick:2 @@',
+  '&& running a task scheduled for 2. &&',
+  'vote outcome: {"changeParam":{"key":"main","parameterName":"Electorate"},"proposedValue":{"brand":"[Alleged: Zoe Invitation brand]","value":[{"description":"questionPoser","expiry":"[undefined]","fee":"[undefined]","handle":"[Alleged: InvitationHandle]","installation":"[Alleged: Installation]","instance":"[Alleged: InstanceHandle]","zoeTimeAuthority":"[undefined]"}]}}',
+  'updated to {"brand":"[Alleged: Zoe Invitation brand]","value":[{"description":"questionPoser","expiry":"[undefined]","fee":"[undefined]","handle":"[Alleged: InvitationHandle]","installation":"[Alleged: Installation]","instance":"[Alleged: InstanceHandle]","zoeTimeAuthority":"[undefined]"}]}',
+  'Validation complete: true',
+  '@@ schedule task for:4, currently: 2 @@',
+  'Voter Alice voted for {"changeParam":{"key":"main","parameterName":"MalleableNumber"},"proposedValue":"[299792458n]"}',
+  'Voter Bob voted for {"changeParam":{"key":"main","parameterName":"MalleableNumber"},"proposedValue":"[299792458n]"}',
+  'Voter Carol voted for {"noChange":{"key":"main","parameterName":"MalleableNumber"}}',
+  'Voter Dave voted for {"changeParam":{"key":"main","parameterName":"MalleableNumber"},"proposedValue":"[299792458n]"}',
+  'Voter Emma voted for {"noChange":{"key":"main","parameterName":"MalleableNumber"}}',
+  '@@ tick:3 @@',
+  '@@ tick:4 @@',
+  '&& running a task scheduled for 4. &&',
+  'vote outcome: {"changeParam":{"key":"main","parameterName":"MalleableNumber"},"proposedValue":"[299792458n]"}',
+  'updated to "[299792458n]"',
+  'MalleableNumber was changed to "[602214090000000000000000n]"',
+  'current value of MalleableNumber is 299792458',
+  'Electorate was changed to {"brand":"[Alleged: Zoe Invitation brand]","value":[{"description":"questionPoser","expiry":"[undefined]","fee":"[undefined]","handle":"[Alleged: InvitationHandle]","installation":"[Alleged: Installation]","instance":"[Alleged: InstanceHandle]","zoeTimeAuthority":"[undefined]"}]}',
+  'Electorate was changed to {"brand":"[Alleged: Zoe Invitation brand]","value":[{"description":"questionPoser","expiry":"[undefined]","fee":"[undefined]","handle":"[Alleged: InvitationHandle]","installation":"[Alleged: Installation]","instance":"[Alleged: InstanceHandle]","zoeTimeAuthority":"[undefined]"}]}',
```

Comment:
Two of these... hm... one is a consequence of setting the electorate initially and the other is the result of the vote?